incubator/elasticsearch/values.yaml

```diff
@@ -11,6 +11,12 @@ image:
   # pullSecrets:
   #   - registry-secret
 
+busybox:
```

Comment:
"busybox" is maybe not the best name - it could potentially be any image - maybe initImage?
no strong opinions.
But could you rebase the branch and I can have a second look at it?

Reply:
Ok, PR changed as you suggest it